packages/storage-plus/src/lib.rs

```diff
@@ -8,25 +8,29 @@ mod item;
 mod iter_helpers;
 mod keys;
 mod map;
+mod multi_index;
 mod path;
 mod prefix;
 mod snapshot;
+mod unique_index;
 
 pub use endian::Endian;
 #[cfg(feature = "iterator")]
 pub use indexed_map::{IndexList, IndexedMap};
 #[cfg(feature = "iterator")]
 pub use indexed_snapshot::IndexedSnapshotMap;
 #[cfg(feature = "iterator")]
-pub use indexes::{
-    index_string, index_string_tuple, index_triple, index_tuple, Index, MultiIndex, UniqueIndex,
-};
+pub use indexes::{index_string, index_string_tuple, index_triple, index_tuple, Index};
 pub use item::Item;
 pub use keys::{I128Key, I16Key, I32Key, I64Key, I8Key};
 pub use keys::{Prefixer, PrimaryKey, U128Key, U16Key, U32Key, U64Key, U8Key};
 pub use map::Map;
+#[cfg(feature = "iterator")]
+pub use multi_index::MultiIndex;
```

Comment:
If we pull them out (which I agree with), how about a module for them all.

eg. indexes/mod.rs, indexes/unique.rs, indexes/multi.rs

And then we can keep the top-level export them same even, just handle differently internally.
(There are so many files in storage-plus I am not in favour of sub-modules when possible)

Reply:
Like with `snapshot`, I agree, let's do it in a follow-up

Reply:
Done in #548.